systemd-swap

```diff
@@ -70,7 +70,18 @@ get_fs_type(){
   elif [[ -d $(dirname "$1") ]]; then path=$(dirname "$1");
   else ERRO "swapfc_path is invalid"
   fi
-  df "$path" --output=fstype | tail -n 1
+
+  fstype=$(df "$path" --output=fstype | tail -n 1)
+
+  if [ "$fstype" == "-" ]; then
```

Comment:
`[[` is a bit faster, use that instead of `[`. See http://web.archive.org/web/20120311112022/http://bashcurescancer.com/which-comparator-test-bracket-or-double-bracket-is-fastest.html (you can test it for yourself)